src/codec/h264/parser.rs

```diff
@@ -266,6 +266,10 @@ pub struct SliceHeader {
     /// the bottom field of a coded frame specified in clause 8.2.1.
     pub delta_pic_order_cnt: [i32; 2],
 
+    /// This value is required by V4L2 stateless decode params so it is calculated
```

Comment:
Makes sense. If it is only needed for V4L2 purpose, it might not be a right place to calculate here in the first place. My todo to check and move this if relevant.